src/runtime/runtime.js

```diff
@@ -535,9 +539,6 @@ export class ConfiguredRuntime {
     /** @private @const {!Logger} */
     this.logger_ = new Logger(this);
 
-    /** @private @const {!AnalyticsService} */
```

Comment:
There's a big comment regarding the usage of "this" as a parameter.  It is currently:

    //NOTE: 'this' is passed in as a DepsDef.  Do not pass in 'this' before
    //analytics service and entitlements manager are constructed unless
    //you are certain they do not rely on them because they are part of that
    //definition.

It should minimally be before the first usage of this as a parameter.  

Not caused by you but you might want to consider how to make it more likely that a developer will actually read the comment too (or clean up the language) as the paradigm of passing "this" around as something that isn't really completely built yet seems quite delicate to me.

Reply:
ACK. Thank you for pointing it out. I'm open to a better way.

Reply:
It doesn't look like you moved the comment above where you are instantiating analytics service.

Reply:
Comment has been addressed in the new PR.